openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -702,20 +702,24 @@ def setUp(self):
     def _assert_supplemented(self, actual, **kwargs):
         """DRY helper used to verify that program data is extended correctly."""
         course_overview = CourseOverview.get_from_id(self.course.id)  # pylint: disable=no-member
-
         run_mode = dict(
             factories.RunMode(
                 course_key=unicode(self.course.id),  # pylint: disable=no-member
                 course_url=reverse('course_root', args=[self.course.id]),  # pylint: disable=no-member
                 course_image_url=course_overview.course_image_url,
                 start_date=self.course.start.strftime(self.human_friendly_format),
                 end_date=self.course.end.strftime(self.human_friendly_format),
+                is_course_ended=self.course.end < timezone.now(),
                 is_enrolled=False,
                 is_enrollment_open=True,
                 marketing_url='',
             ),
             **kwargs
         )
+
+        if not run_mode.get('is_enrollment_open', False) and course_overview.enrollment_start:
+            run_mode['enrollment_open_date'] = course_overview.enrollment_start.strftime(self.human_friendly_format)
```

Comment:
@schenedx your updates to `test_course_enrollment_status` are what I was looking for. Since you've made those updates, these lines here are redundant. Please remove them.